lib/stripe/card_cloner.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Here we clone
+# - a card (card_*) stored in a customer in a platform account
+# into
+# - a payment method (pm_*) in a new customer in a connected account
+#
+# This process is used in the migration between using the Stripe Charges API (stripe_connect)
+#   and the Stripe Payment Intents API (stripe_sca)
+#
+# This process can be deleted once all hubs are running on the new stripe_sca method and all cards in the system have been migrated to the new payment_methods
```

Comment:
I'm reviewing this on Github and need to scroll horizontally which is a bit awkward. Can I have shorter lines, please?

Reply:
yes, this was fixed in later commits :+1: